### ISSUE TITLE
test.mock_socket.socket is missing

### ISSUE THREAD

**Issue by LeonarddeR** [astral-sh/python-build-standalone#589]
We're investigating using uv in [NVDA](https://github.com/nvaccess/nvda/). We have a test that uses `test.mock_socket.socket`, but unfortunately that's missing from these redistributable python builds.

**Comment by LeonarddeR** [comment]
Never mind, we just should avoid using the internal test module.
